Makefile

```diff
@@ -1,7 +1,16 @@
-.PHONY: test
+.DEFAULT_GOAL := bin/ssm-env
+
+ARGS :=
+
+version := $(shell git describe --tags --abbrev=0)
```

Comment:
Does this work when building a commit that isn't tagged?

Reply:
Yes, it uses the most recent tag, but I've now updated it to remove the --abbrev, which will include the number of commits since the tag and the current sha as well (pretty nifty: `v0.0.5-2-g4069de5`). On the tagged commit itself, it'll be clean and just the version.